rebasehelper/output_tool.py

```diff
@@ -70,21 +70,25 @@ def print_cli_summary(cls, app):
                 ConsoleHelper.cprint('\nABI changes occured. Check abipkgdiff output', color='red')
 
         ConsoleHelper.cprint('\nAvailable logs:', color='yellow')
-        print('{0}:\n{1}'.format('Debug log', app.debug_log_file))
+        print('{0}:\n{1}'.format('Debug log', cls.get_relative_results_path(os.path.relpath(app.debug_log_file,
+                                                                                            app.results_dir))))
```

Comment:
Wouldn't it make more sense to incorporate `os.path.relpath(x, app.results_dir)` into `get_relative_results_path()`?

Reply:
Well the paths used for `os.path.relpath` are supposed to be similiar, but often times only relative path is given. Currently I do not see how we could benefit from using it in `get_relative_results_path`

Reply:
In that case I think the method should be renamed. `get_relative_results_path()` implies that is converts its arguments to a path relative to something, which is not exactly true.